bin/upload-staticfiles.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -exo pipefail
+
+source docker/bin/set_git_env_vars.sh
+
+if [[ "$BRANCH_NAME" != "prod" ]]; then
+    # nothing to do if not prod deployment
+    exit 0
+fi
+
+CONTAINER_NAME="bedrock-${BRANCH_AND_COMMIT}"
+
+rm -rf static
+# have to rerun staticfiles without symlinks or we just copy broken symlinks
+docker-compose run --name "$CONTAINER_NAME" release docker/bin/build_staticfiles.sh --nolink
+docker cp "${CONTAINER_NAME}:/app/static" ./static
+docker rm -f "$CONTAINER_NAME"
+
+cd static
+for BUCKET in stage prod; do
+    # hashed filenames
+    aws s3 sync \
+        --exclude "*" \
+        --include "*.????????????.*" \
+        --acl public-read \
+        --cache-control "max-age=315360000, public, immutable" \
+        --profile bedrock-media \
+        . "s3://bedrock-${BUCKET}-media/"
+    # non-hashed filenames
+    # may not need to include non-hashed files
+    # TODO look into this if this is slow or makes the bucket too large
```

Comment:
mdn syncs required an [additional flag](https://docs.aws.amazon.com/cli/latest/reference/s3/sync.html) `--page-size` to help prevent timeouts on large buckets.

Reply:
according to the docs:

```
--page-size (integer) The number of results to return in each response to a list operation. The default value is 1000 (the maximum allowed). Using a lower value may help if an operation times out.
```

To my knowledge we're not doing any list operations. Or is it doing that behind-the-scenes during the sync?

Reply:
I seem to remember troubleshooting the MDN sync and seeing that `aws s3 sync` used list to compare files.

Reply:
@metadave do you recall to what value they set `--page-size`?

Reply:
[100](https://github.com/mdn/infra/blob/master/apps/mdn/utils/mdn_backup_cron/image/mdn_sync.sh#L11)

Reply:
definitely not required, but keep it in mind in case there are sync issues.